GitUI/Themes/bright.css

```diff
@@ -40,3 +40,8 @@
 .DiffRemovedExtra { color: #ffa5a5 }
```

Comment:
Why keep both bright.css and invariant.css? I do not see bright.css used?

Reply:
"invariant.css" is not the theme which gets applied if the user chooses "default" theme, but rather a technical theme, user cannot choose it. The purpose for invariant theme is to provide a fixed set of colors, to compare against other themes.

When user choses "default" theme, GitExtensions will not apply any hacks to tweak colors, so we are guaranteed to get what we see in WinForms designer. As a result the "default" theme cannot have a colorblind variation.

Therefore to provide a colorblind variation of non-dark theme, we need to introduce a non-default theme, which user can choose and then have the option to check "Colorblind" flag.

Reply:
> I do not see bright.css used?

bright.css can be chosen in Theme menu

Reply:
The file is not included with the installer (should it be somewhere else too?)
otherwise OK

Reply:
You are right, I forgot it, thanks for the vigor!

Reply:
> What I am more interested is supporting import to update just some values (to support for example #8432) and make theme creation or maintenance easier.

Good point, feel free to add your commit to this PR so our changes get to be reviewed and delivered as 1 package

Reply:
>Good point, feel free to add your commit to this PR so our changes get to be reviewed and delivered as 1 package

done.
I have also added some comments  in the css files to explain a little more how to create or modify a theme.
Feel free all to correct my bad English sentences...

Reply:
Thank you

Reply:
> Feel free all to correct my bad English sentences...

Says one non-english speakers to another. We are blessed people, for native english the code is like:
```
si (fenêtre.estUverte()) {
  retourner vérité;
} sinon {
  retourner faux;
}
```